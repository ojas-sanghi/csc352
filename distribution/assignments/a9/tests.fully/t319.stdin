25
58
+
15
6
*
*
