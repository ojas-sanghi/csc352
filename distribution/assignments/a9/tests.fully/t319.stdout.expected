25 58 + 15 6 * *
((25+58)*(15*6))
